src/Client.php

```diff
@@ -303,9 +303,9 @@ public function getAccountInfo(): array
      *
      * @return array
      */
-    public function revokeToken(): array
+    public function revokeToken()
     {
-        return $this->rpcEndpointRequest('auth/token/revoke', []);
+        return $this->voidEndpointRequest('auth/token/revoke');
```

Comment:
@freekmurze I personally prefer two different methods when have different return types, but if you prefer I can implement the other way.